# HG changeset patch
# User Jon Dufresne <jon.dufresne@gmail.com>
# Date 1521133853 25200
# Node ID c938f5b558833dd290bbdcbde26c97a6bbfea623
# Parent  8784bdea1cb29b2a5b5d21045b024880ab84c05d
Unify Python 2 and 3 dependency on python-ldap >= 3.0

python-ldap now supports both Python 2 and 3. The pyldap package is
deprecated. The project recommends switching to python-ldap instead.
From the github page:

https://github.com/pyldap/pyldap

> THIS FORK IS DEPRECATED
>
> The pyldap fork was merged back into python-ldap, and released as
> python-ldap 3.0.0b1.

diff --git a/README b/README
--- a/README
+++ b/README
@@ -1,6 +1,6 @@
-This project provides a mock replacement for python-ldap (pyldap on Python
-3). It's useful for any project that would like to write unit tests against
-LDAP code without relying on a running LDAP server.
+This project provides a mock replacement for python-ldap. It's useful for any
+project that would like to write unit tests against LDAP code without relying
+on a running LDAP server.
 
 * Repository: https://bitbucket.org/psagers/mockldap
 * Documentation: https://mockldap.readthedocs.io/
diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -40,11 +40,8 @@
     keywords=['mock', 'ldap'],
     install_requires=[
         'funcparserlib == 0.3.6',
-
-        'pyldap; python_version >= "3.0"',
-        'python-ldap >= 2.0; python_version < "3.0"',
-
         'mock; python_version < "3.0"',
+        'python-ldap >= 3.0',
     ],
     setup_requires=[
         'setuptools >= 0.6c11',
